src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts

```diff
@@ -1921,56 +1921,56 @@
     <message>
         <location filename="../pages/page_soundtheme.ui" line="14"/>
         <source>Form</source>
-        <translation type="unfinished">Formulär</translation>
+        <translation>Formulär">Formulär</translation>
     </message>
     <message>
         <location filename="../pages/page_soundtheme.ui" line="22"/>
         <location filename="../pages/page_soundtheme.ui" line="67"/>
         <location filename="../pages/page_soundtheme.ui" line="112"/>
         <source>Enabled</source>
-        <translation type="unfinished"></translation>
+        <translation>Aktiverad</translation>
     </message>
     <message>
         <location filename="../pages/page_soundtheme.ui" line="36"/>
         <location filename="../pages/page_soundtheme.ui" line="81"/>
         <location filename="../pages/page_soundtheme.ui" line="126"/>
         <source>TextLabel</source>
-        <translation type="unfinished"></translation>
+        <translation>unfinished</translation>
```

Comment:
This needs to be finished, or reverted back to having the `type="unfinished"` tag.